hudi-client/hudi-client-common/src/main/resources/log4j2.properties

```diff
@@ -15,9 +15,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###
-log4j.rootLogger=INFO, A1
-# A1 is set to be a ConsoleAppender.
-log4j.appender.A1=org.apache.log4j.ConsoleAppender
-# A1 uses PatternLayout.
-log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+status = warn
+name = HudiConsoleLog
```

Comment:
We need to clarify why we have this config in production (not test) resources.

If we're previously trying to influence configuration of users of the Spark client, then we can't upgrade this one to log4j2 since there are still clients using older Spark versions using log4j1

Reply:
@alexeykudinkin is this something that you're going to take on? I don't have much context here

Reply:
I don't have that context either unfortunately. Let's sync up on Slack regarding the next steps.

Reply:
it anyway looks like an anti-pattern to bundle production log properties. we should remove it and mark it as a migration item for next major release, if it breaks anything.